kong/plugins/hmac-auth/access.lua

```diff
@@ -79,30 +84,31 @@ local function create_hash(request, hmac_params, headers)
       signing_string = signing_string.."\n"
     end
   end
+
   return ngx_sha1(hmac_params.secret, signing_string)
 end
 
-local function is_digest_equal(digest_1, digest_2)
+local function is_signature_equal(signature_1, signature_2)
```

Comment:
After this change, `digest` has a separate role: To signify Sha256 hash of request body. I renamed this function and its variable to avoid any confusion in future. Also, imo the function name should sync with its functionality, it is equating signatures so its name should justify this.